lib/solidus_support.rb

```diff
@@ -3,6 +3,7 @@
 require 'solidus_support/version'
 require 'solidus_support/migration'
 require 'solidus_support/engine_extensions'
+require 'solidus_support/legacy_event_compat'
```

Comment:
We are using `SolidusSupport::LegacyEventCompat` in `lib/solidus_support/engine_extensions`. Wouldn't be more correct to require this before that file?

Reply:
Sure! Done.